db/character_skill_level_to_effects_junctions_tables/zzz_daemonslayer_skills.tsv

```diff
@@ -0,0 +1,4 @@
+character_skill_key	effect_key	level	effect_scope	value
```

Comment:
Oops, looks like you didn't clean up (i.e. remove) the files from the first commit so there are 3 duplicate entries/duplicate dbs

Reply:
It's closed!

Reply:
Oh yeah sorry i noticed what you meant now, had to do some cleanup